private/src/scripts/editor/blocks/post-list/components/display/LinkList.jsx

```diff
@@ -1,30 +1,36 @@
-const LinkItem = (props) => (
-  <li>
-    <article className="linkList-item">
-      {props.tag && (
-        <span className="linkList-itemMeta">
-          <a href="#">{props.tag.title}</a>
-        </span>
-      )}
-      <h3 className="linkList-itemTitle">
-        <a>{props.title}</a>
-      </h3>
-      <div className="postInfo-container">
-        {props.showPostDate && (
-          <p className="linkList-itemDate">
-            <span className="dateTerm">Date: </span>
-            <span className="dateDescription">{props.date}</span>
-          </p>
-        )}
-        {props.showAuthor && (
-          <p className="linkList-itemAuthor">
-            <span className="authorTerm">Author: </span>
-            <span className="authorDescription">{props.authorName}</span>
-          </p>
+const LinkItem = (props) => {
+  const backgroundImage = props.featured_image
+    ? { backgroundImage: `url(${props.featured_image})` }
+    : '';
+
+  return (
+    <li>
+      <article className="linkList-item" style={{ ...backgroundImage }}>
```

Comment:
```suggestion
  const itemStyles = {};
  
  if (props.featured_image) {
    itemStyles.backgroundImage = `url('${props.featured_image}')`;
  }

  return (
    <li>
      <article className="linkList-item" style={itemStyles}>
```

This will allow for a cleaner diff if we extend the styles with additional properties later.

Reply:
Resolved